docs/devel/README.md

```diff
@@ -103,16 +109,16 @@ git clone https://gitlab.com/CentOS/automotive/sample-images.git
 git submodule update --init
 cd sample-images/
 rm -rf _build
-rm -f cs9-qemu-qmcontainer-regular.x86_64.qcow2
-rm -f cs9-qemu-qmcontainer-ostree.x86_64.qcow2
-./build --distro cs9 --target qemu --define 'extra_repos=[{\"id\":\"local\",\"baseurl\":\"file:///root/rpmbuild/RPMS/noarch\"}]' --define 'extra_rpms=[\"qm-1.0\",\"vim-enhanced\",\"strace\",\"dnf\",\"gdb\",\"polkit\",\"rsync\",\"python3\",\"openssh-server\",\"openssh-clients\"]' --define 'ssh_permit_root_login=true' --define 'ssh_permit_password_auth=true' cs9-qemu-qmcontainer-regular.x86_64.qcow2
+rm -f cs9-qemu-qm-container-regular.x86_64.qcow2
+rm -f cs9-qemu-qm-container-ostree.x86_64.qcow2
+./build --define 'extra_repos=[{"id":"local","baseurl":"file:///root/rpmbuild/RPMS/noarch"}]' --define 'extra_rpms=["qm-1.0", "vim-enhanced", "strace", "dnf", "gdb", "polkit", "rsync", "python3", "openssh-server", "openssh-clients"]' --define 'ssh_permit_root_login=true' --define 'ssh_permit_password_auth=true' cs9-qemu-qm-container-regular.x86_64.qcow2
```

Comment:
It is about time to work with 

bin/bash auto-image-builder.sh build --export qcow2  --define 'extra_rpms=["audit","dnf"]'  custom-images/complex.aib.yml auto-osbuild-qemu-autosd9-aib-ostree.x86_66.qcow2


please mentions those two also
./images/simple-qm-container.aib.yml
./images/simple-qm.aib.yml

With documentation reference to upstream docs